### DISCUSSION TITLE
How to deal with very very large dataset?

### DISCUSSION THREAD

**Discussion by b-y-f** [blue-yonder/tsfresh#1004] [Q&A] [answered]
we have id 1-80,000 and each id is associated with 30,000 rows of data. I guess this will take ages to finish with tsfresh

**Comment by nils-braun** [comment, marked as answer]
I have given some advice in the issue you created https://github.com/blue-yonder/tsfresh/issues/1005.
In short: use another extractor setting, chunk up the data into windows (because some extractors do not scale linearly with the number of rows) or use a computing cluster/batch system.
Let's continue the discussion on the issue.
